docs/install-gpu-pip-windows.rst

```diff
@@ -1,6 +1,6 @@
 .. code-block:: bash
 
-    pip3 install -U pip
-    pip3 install -U setuptools wheel
+    pip install -U pip
+    pip install -U setuptools wheel
 
-    pip3 install --pre autogluon
+    pip install --pre autogluon
```

Comment:
Sanity check: do we want `--pre` here?

Reply:
Yes. --pre is removed only in the `v0.x.y` and `stable` branches, not in master.